simulator/host_datastore_browser.go

```diff
@@ -128,12 +128,13 @@ func (s *searchDatastore) queryMatch(file os.FileInfo) bool {
 			}
 		case *types.VmDiskFileQuery:
 			if ext == ".vmdk" {
-				if strings.HasSuffix(name, "-flat.vmdk") {
-					// only matches the descriptor, not the backing file(s)
-					return false
-				}
+				// if strings.HasSuffix(name, "-flat.vmdk") {
```

Comment:
Can we just remove this commented out code?

Reply:
sure